src/v0/destinations/klaviyo/transform.js

```diff
@@ -312,7 +336,7 @@ const processRouterDest = async (inputs, reqMetadata) => {
           // if not transformed
           getEventChunks(
             {
-              message: await process(event),
+              message: await process(event, reqMetadata),
```

Comment:
Since we are already forming a Map type structure in process lets pass it  into getEventChunks function it will be easier to segregate

Reply:
@utsabc , we are already segregating at process level, one more thing is all the calls are using this function for segregation. we can take it as improvement. let's not increase the scope as of now for this task